examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -19,26 +18,33 @@
     build_airflow_polling_sensor,
 )
 from dagster_airlift.core.serialization.compute import _metadata_key, compute_serialized_data
+from dagster_airlift.core.state_backed_defs_loader import StateBackedDefinitionsLoader
 
 
-def get_or_compute_airflow_data(
-    *, airflow_instance: AirflowInstance, defs: Definitions
-) -> AirflowDefinitionsData:
-    context = DefinitionsLoadContext.get()
-    metadata_key = _metadata_key(airflow_instance.name)
-    if (
-        context.load_type == DefinitionsLoadType.RECONSTRUCTION
-        and metadata_key in context.reconstruction_metadata
-    ):
-        airflow_definition_data = deserialize_value(
-            context.reconstruction_metadata[metadata_key], as_type=AirflowDefinitionsData
-        )
-        assert isinstance(airflow_definition_data, AirflowDefinitionsData)
-        return airflow_definition_data
-    else:
-        serialized_data = compute_serialized_data(airflow_instance=airflow_instance, defs=defs)
+@dataclass
+class AirflowInstanceDefsLoader(StateBackedDefinitionsLoader[AirflowDefinitionsData]):
+    airflow_instance: AirflowInstance
+    explicit_defs: Definitions
```

Comment:
@dpeng817 want to call out this terminology change. Kept on getting the defs passed in and the defs programmatically generated so moved this to `explicit_defs` to indicate these are the ones that the user passed in.